applications/CoSimulationApplication/python_scripts/data_transfer_operators/sum_distributed_to_distributed.py

```diff
@@ -0,0 +1,43 @@
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_data_transfer_operator import CoSimulationDataTransferOperator
+
+# Other imports
+import numpy as np
+
+def Create(settings):
+    return SumDistributedToDistributed(settings)
+
+class SumDistributedToDistributed(CoSimulationDataTransferOperator):
+    """DataTransferOperator to sum values on one interface and put it to another interface, if desired equally distributed.    
+    """
+    def _ExecuteTransferData(self, from_solver_data, to_solver_data, transfer_options):        
+        if not to_solver_data.IsDefinedOnThisRank():
+            return
+
+        data_array = from_solver_data.GetData()
+
+        value = data_array.sum()
+        value = from_solver_data.GetModelPart().GetCommunicator().GetDataCommunicator().SumAll(value)     
+
+        to_solver_values = to_solver_data.GetData()
+        to_solver_values.fill(value)
```

Comment:
are you sure this is correct?
This would increase the total amount of whatever you exchange

On one side you sum all, then on the other side you set this value to EVERY entity

Reply:
For my case I used the transfer_operator "distribute_values" to distribute the sum equally to all entities.

In this case the sum of both sides is equal.

Reply:
ok thats what I thought

Can you please explain this clearly in the docstring to avoid confusion?

Reply:
Edited the file after   #9472